factfinder/data/acs/2010/median.json

```diff
@@ -702,6 +720,8 @@
     },
     "mdgr": {
         "design_factor": 1.6,
+        "top_coding": false,
```

Comment:
Again, I believe this should be true, but it looks like the top coding threshold is not the lower boundary of the top bin. I'm not sure how to implement this one.